packages/clis/parser/game-files/sector-parser.ts

```diff
@@ -410,8 +411,6 @@ const SimpleItemStruct = {
     vegetationSpheres,
     quadInfo,
   },
-  // useful for routing?
-  // https://modding.scssoft.com/wiki/New_Editor_Features_info_-_old_%2B_1.47
```

Comment:
I thought it was... until I learned "trajectory" is used for special transport deliveries 🫠